src/app.js

```diff
@@ -1,12 +1,14 @@
 require('dotenv').config();
 const bodyParser = require('body-parser');
 const express = require('express');
-const octokit = require('@octokit/rest');
+
+// GitHub official REST API client for Node.js
+const Octokit = require('@octokit/rest');
 const nacl = require('tweetnacl');
 nacl.util = require('tweetnacl-util');
 
-const username = 'yourusername';  // TODO: your GitHub username here
-const github = new octokit({ debug: true });
+const username = 'cmoore4os'; // process.env.GITHUB_USERNAME; // TODO: your GitHub username here
```

Comment:
Good thinking on acknowledging the use of an environment variable to store this!